indexer.go

```diff
@@ -132,7 +165,7 @@ func CleanupIndexes(url string, alias string) error {
 func MakeJSONRequest(method string, url string, body string, jsonStruct interface{}) (*http.Response, error) {
 	req, _ := http.NewRequest(method, url, bytes.NewReader([]byte(body)))
 	req.Header.Add("Content-Type", "application/json")
-	resp, err := http.DefaultClient.Do(req)
+	resp, err := httpx.Do(http.DefaultClient, req, ElasticRetries(), nil)
```

Comment:
but now we're back to creating the retry config for every request 😄 

my thinking was one global var called `retryConfig`, initialize that in `init()` and use it for all requests